lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -443,6 +447,37 @@ bool isAttributeEditAllowed(const UsdPrim& prim, const TfToken& attrName)
     return true;
 }
 
+bool isEditTargetLayerModifiable(const PXR_NS::UsdStageWeakPtr stage, std::string* errMsg)
+{
+    const auto editTarget = stage->GetEditTarget();
+    const auto editLayer = editTarget.GetLayer();
+
+    if (editLayer && !editLayer->PermissionToEdit()) {
+        if (errMsg) {
+            std::string err = TfStringPrintf(
+                "Cannot edit the layer [%s] because it is locked (PermissionToEdit is false).",
```

Comment:
Super minor nit-pick: would word the message as "Cannot edit layer [%s] because it is locked (PermissionToEdit is false)."